index.html

```diff
@@ -0,0 +1,221 @@
+---
+{
+"altLangPage": "index-fr.html",
+"dateModified": "2025-01-22",
+"description": "Office of the Auditor General of Canada home page template",
+"language": "en",
+"title": "Office of the Auditor General of Canada",
+"layout": home,
+"pageclass": "oag-theme",
+"noFooterMain":true
+}
+---
+<div class="pt-4 pb-4 bg-center bg-cover"
+	data-bgimg-srcset="src/assets/dark_background2.png 600w, src/assets/dark_background2.png 992w">
+	<section class="container">
+		<div class="row">
+			<div class="col-xs-12">
+				<h2 class="mrgn-bttm-xl text-white">We serve Canada through leadership and partnerships in audits that
+					support trust in public
+					institutions and continued public service excellence.</h2>
+			</div>
+		</div>
+		<div class="row wb-eqht hght-inhrt">
+			<!-- Left Column -->
+			<div class="col-xs-12 col-sm-7">
+				<div class="bg-center bg-cover well hght-inhrt pt-3 pl-0"
+					data-bgimg-srcset="src/assets/bear-background.png 600w, src/assets/bear-background.png 992w">
+					<div class="p-3 ">
+						<span class="font-weight-bold p-1 label label-primary ">Reports of the Auditor General of
+							Canada</span>
+						<section class="alert alert-info mrgn-tp-lg mrgn-bttm-lg">
+							<p>3 Upcoming Reports:</p>
+							<time class="h2 text-primary" datetime="2024-11-18">November 18, 2024</time>
+						</section>
+						<a class="btn btn-lg btn-primary mt-3 " href="#">See all upcoming reports</a>
+					</div>
+				</div>
+			</div>
+			<!-- Right Column -->
+			<div class="col-xs-12 col-sm-5">
+				<div class="bg-white well pb-5">
+					<p class="mb-1">
+						<span class="label label-primary small font-weight-bold p-1">AG Reports</span>
+						<span>May 6, 2024</span>
+					</p>
+					<p class="mb-3">Strategic Innovation Fund’s Net Zero Accelerator Initiative—Innovation,
+						Science and Economic Development Canada</p>
+					<a href="#" class="pull-right no-undrln text-primary">Read report &rarr;</a>
+				</div>
+				<div class="bg-white well pb-5 mb-0">
+					<p class="mb-1">
+						<span class="label label-primary small font-weight-bold p-1">CESD Reports</span>
+						<span>Feb. 18, 2024</span>
+					</p>
+					<p class="mb-3">Housing in First Nations Communities</p>
+					<a href="#" class="pull-right no-undrln text-primary">Read report &rarr;</a>
+				</div>
+			</div>
+		</div>
+	</section>
+
+</div>
+
+<section class="mrgn-tp-lg mrgn-bttm-xl">
+	<h2 class="wb-inv">Features</h2>
+	<div class="container">
+		<div class="row wb-eqht-grd">
+			<!-- Feature 1 -->
+			<div class="col-sm-4">
+				<div class="panel panel-default gc-hvr-shadow eqht-trgt hght-inhrt">
```

Comment:
Leverage the Service and info component

And use the stretchlink to extend the scope of the clickable area